src/Common/src/CoreLib/System/Resources/ResourceReader.cs

```diff
@@ -133,7 +135,13 @@ public ResourceReader(string fileName)
             }
         }
 
-        public ResourceReader(Stream stream)
+        public
+#if RESOURCES_EXTENSIONS
```

Comment:
>f  [](start = 2, length = 2)

nit: for readability, I would write the whole constructor signature in the #if #else blocks. 

#if RESOURCES_EXTENSIONS
       public DeserializingResourceReader(Stream stream)
#else
        public ResourceReader(Stream stream)
#endif

Reply:
Fair, I had it that way in a previous commit but then changed it because I didn't want to duplicate the parameters (DRY).  I'm ok changing it though.